test/integration/015_cli_invocation_tests/test_cli_invocation.py

```diff
@@ -111,3 +112,31 @@ def test_postgres_toplevel_dbt_run_with_profile_dir_arg(self):
         # make sure the test runs against `custom_schema`
         for test_result in res:
             self.assertTrue(self.custom_schema, test_result.node.injected_sql)
+
+
+class TestCLIInvocationWithProjectDir(ModelCopyingIntegrationTest):
+
+    @property
+    def schema(self):
+        return "test_cli_invocation_015"
+
+    @property
+    def models(self):
+        return "models"
+
+    @use_profile('postgres')
+    def test_postgres_dbt_commands_with_cwd_as_project_dir(self):
+        self._run_simple_dbt_commands(os.getcwd())
+
+    @use_profile('postgres')
+    def test_postgres_dbt_commands_with_randomdir_as_project_dir(self):
+        workdir = os.getcwd()
+        with tempfile.TemporaryDirectory() as tmpdir:
+            os.chdir(tmpdir)
+            self._run_simple_dbt_commands(workdir)
```

Comment:
I think we will need an `os.chdir(workdir)` after this line for Windows tests. While linux/osx will happily `rm -r` the directory you're in for the most part, Windows gets cranky about it.

Reply:
That makes sense, changes done.

and also curious: would this be caught in tests (i.e. does dbt run tests w/ windows) test would fail?

Reply:
Yup! I'm about to kick them off now, I just didn't want to run the full test suite and then have to re-run later. Running tests for external contributors isn't quite as friction-free as we'd like yet.

Reply:
I see HAHA, that's awesome, I assume it's a lot of resources to use to run all these tests 😛  Appreciate it.